newm-server/src/test/kotlin/io/newm/server/features/distribution/eveara/EvearaDistributionRepositoryTest.kt

```diff
@@ -153,7 +153,10 @@ class EvearaDistributionRepositoryTest : BaseApplicationTests() {
                 album = "Daisuke",
                 duration = 200000,
                 track = 1,
-                copyright = "© Mirai Music Publishing, ℗ Danketsu, Mirai Music, NSTASIA",
+                compositionCopyrightOwner = "© Mirai Music Publishing",
```

Comment:
I'm not sure if this is how we want to do it.  I think the final order should be

`<symbol> <year> <owner>`

If the frontend is sending us the symbol, this is weird to inject the year in the middle of the other string.